rwengine/src/render/MapRenderer.cpp

```diff
@@ -219,12 +220,18 @@ void MapRenderer::draw(GameWorld* world, const MapInfo& mi)
         (rgbaValue >> 24) / 255.0f,
         ((rgbaValue >> 16) & 0xFF) / 255.0f,
         ((rgbaValue >> 8) & 0xFF) / 255.0f,
-        1.0f // Note: Alpha is ignored
+        1.0f // Note: Alpha is not controlled by blip
       );
 
+      // @todo This is a hack which was introduced because GTA uses a 16px radar_north at 640x480
+      //       OpenRW uses 24 instead, which means we have a 150% increase in scale somehow..
+      float badScale = 1.5f;
```

Comment:
apparently this is just really small on higher res.. probably size given in pixels?
I should remove this factor and keep it that way